build.gradle

```
@@ -186,6 +186,15 @@ tasks.register('createReleaseTag', CreateGitTag) {
 
 tasks.named("githubRelease") {
     dependsOn("createReleaseTag")
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
+}
+
+tasks.named('publishPlugins') {
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
+}
+
+tasks.withType(Sign) {
```

This would force the configuration of all `Sign` tasks in all builds. The following would keep the configuration lazy:
```suggestion
tasks.withType(Sign).configureEach {
```

---

Thanks, I've made the change.